### PR TITLE
Ejinks

### PR DESCRIPTION
**Pull request by ejinks**

### CODE REVIEW

File: views/partials/comment.handlebars [hunk 1]
```diff
@@ -1,30 +1,41 @@
...
+                <div id="commentEditError-{{_id}}" hidden></div>
+            </div>
+            <div id="commentDelete">
+                <form action="/libraries/{{@root.libraryid}}/comments/{{_id}}/delete" method="POST">
+                    <button type="submit">Delete</button>
+                </form>
+            </div>
+        {{/if}}
+        {{#if (eq @root.userid @root.ownerID)}}
+            <div id="commentDelete">
```
Comment (galapatt) [inline views/partials/comment.handlebars:34]: You have two div with the same id "commentDelete"

**Review by galapatt — COMMENTED** [review]
nevermind, all good 👍

**Review by vishwa5854 — APPROVED** [review]

File: views/partials/comment.handlebars (hunk 1 above)
Comment (DanielleDauphinais) [reply views/partials/comment.handlebars:34]: they are in different if statements

**Comment by DanielleDauphinais** [comment]
LGTM

File: views/partials/comment.handlebars (hunk 1 above)
Comment (galapatt) [reply views/partials/comment.handlebars:34]: oops u right. missed that

**Review by galapatt — APPROVED** [review]

**Pull request merged**